.github/ISSUE_TEMPLATE/s2_report.yaml

```diff
@@ -16,82 +16,88 @@ body:
     - type: dropdown
       id: browsers
       attributes:
-          label: What browsers are you seeing the problem in?
+          label: What browser(s) does the issue appear in?
           multiple: true
           options:
               - Firefox
               - Chrome
               - Safari
               - Microsoft Edge
+    - type: textarea
+      id: screenshot
+      attributes:
+          label: Do you have screenshots?
+          description: Attach images by clicking this area to highlight it and then drag files in.
+
     - type: textarea
       id: code
       attributes:
-          label: Sample code or abstract reproduction which illustrates the problem
-          description: Please use [webcomponents.dev](https://studio.webcomponents.dev/workspace/adobe) to abstract the issue in an isolated environment. In most cases, a video or code block is not sufficient enough for us to diagnose the problem.
+          label: Have you reproduced this issue in isolation to illustrate the problem?
+          description: Provide an code sample from [webcomponents.dev](https://studio.webcomponents.dev/workspace/adobe). In most cases, a video or code block is not sufficient enough for us to diagnose the problem.
     - type: textarea
       id: reproduce
       attributes:
-          label: How can we reproduce this issue?
+          label: What are the steps to manually reproduce this issue?
+          description: Provide detailed steps for reproducing the issue.
           value: |
               1. Go to '...'
               2. Click on '....'
               3. Scroll to '....'
               4. Check console
               5. See error
-    - type: dropdown
-      id: severity
-      attributes:
-          label: Severity
-          description: Please select the severity level of this issue. For descriptions of each level, refer to our [documentation](https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md#issue-severity-classification)
-          options:
-              - SEV 1
-              - SEV 2
-              - SEV 3
-              - SEV 4
-              - SEV 5
     - type: textarea
       id: logs
       attributes:
-          label: Logs taken while reproducing problem
-          description: You can attach log files by clicking this area to highlight it and then dragging files in.
+          label: Did you capture any logs while reproducing problem?
+          description: Attach log files by clicking this area to highlight it and then drag files in.
     - type: checkboxes
       id: jira-number
       attributes:
-          label: Jira ticket
-          description: If you would like to track this issue in Jira, check this box and we will comment the Jira ticket ID once the issue has been triaged.
+          label: Would you like to track this issue in Jira?
+          description: We will share the Jira ticket ID in a comment on the issue once it has been triaged.
```

Comment:
@najikahalsema Since this is open source would it make sense to just tag them in the Jira ticket instead of leaving an internal-only comment on the public ticket?

Reply:
I was thinking of just replying "SWC-XXX" in a comment without providing a direct link. We can @ the person in Jira, but that may not account for all cases when the account isn't attached to a name or they don't read their jira email notifications (me)